BlazarUI/app/scripts/models/Log.js

```diff
@@ -14,8 +13,27 @@ class Log extends Model {
   }
 
   url() {
-    return this.logUrl;
+    return `${config.apiRoot}/build/${this.buildNumber}/log/`;
   }
+
+  formatLog(jqxhr) {
+    let cache = [];
+
+    if (jqxhr.status !== 200) {
+      console.warn(jqxhr);
+      return "<p class='x-y-roomy'>Error loading build log. Please check your console for more detail.</p>";
+    }
+
+    const lines = jqxhr.responseJSON.data.split('\n');
+
+    lines.forEach( (line) => {
+      cache.push(`<p class='log-line'>${line}</p>`);
+    });
+
+    return cache.join('\n');
```

Comment:
Let me know if I didn't hear you correctly, I'm not sure if I get what you mean when you say `between p tags` - The `responseJSON.data` returns one long string, so I' using the `\n` to split it, to add the `log-line` class for some css formatting, but I'm also planning on adding links to line numbers in here (like how you can on github, and recently in singularity), so I'll need a place to add that. FWIW, this was a quick pass to get the log showing, I was going to think about it a bit more, but wanted to get something out, so if you had any thoughts on the setup in general, I'm happy to hear.